ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/ActiveStandbyResponse.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import com.google.errorprone.annotations.Immutable;
+import java.util.Map;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Immutable
+public class ActiveStandbyResponse {
+
+  private final ImmutableMap<String, ActiveStandbyEntity> perQueryInfo;
+
+  @JsonCreator
+  public ActiveStandbyResponse(
+      @JsonProperty("perQueryInfo") final Map<String, ActiveStandbyEntity> perQueryInfo) {
```

Comment:
As we talked about in person, you probably will ultimately want the String to be some object which can uniquely define a state store, since the one's we're using at the moment aren't unique.  I guess we can just refactor this once we introduce such an object.

Reply:
Yes, since you are making that change, I will get it once your PR is merged :)